vizro-core/tests/unit/vizro/models/_components/test_button.py

```diff
@@ -86,6 +107,40 @@ def test_button_with_variant(self, variant, expected_color):
         result = vm.Button(variant=variant).build()
         assert_component_equal(
             result,
-            dbc.Button(children="Click me!", href="", target="_top", color=expected_color),
+            dbc.Button(
+                children=html.Span(["Click me!", None], className="button-label"),
+                href="",
+                target="_top",
+                color=expected_color,
+            ),
             keys_to_strip={"id"},
         )
+
+    def test_button_build_tooltip(self):
```

Comment:
Just for the consistency with other similar tests for other models.
```suggestion
    def test_button_build_with_description(self):
```